mobile/src/people/people-company.tsx

```diff
@@ -63,39 +66,46 @@ export class PeopleCompanyImpl extends React.Component<PeopleCompanyProps & Peop
-                        departmentsLists={this.props.departmentLists[index + 1]}
-                        headDepartmentId={head.departmentId}
-                        headDepartmentChiefId={head.chiefId}
-                        focusOnDepartmentWithId={(index + 1) < userFocusedDepartmentsBranch.length ? userFocusedDepartmentsBranch[index + 1].departmentId : null}
-                        currentFocusedDepartmentId={this.props.currentFocusedDepartmentId}
-                        employees={this.props.employees}
-                        key={head.departmentId}
-                        updateDepartmentsBranch={this.props.updateDepartmentsBranch}
-                        requestEmployeesForDepartment={this.props.requestEmployeesForDepartment}
-                        onItemClicked={this.props.onItemClicked}
-                        employeesPredicate={(employee: Employee) => { 
-                            return employee.departmentId === head.departmentId && employee.employeeId !== head.chiefId;
-                        }}
-                    />
-                ))
-            }
-        </ScrollView>;
+        return <View>
+            <SearchPeopleView/>
+            <ScrollView style={{ backgroundColor: '#fff', flex: 1 }}>
+                <EmployeeCardWithAvatar
+                    employee={this.props.employees.employeesById.get(userFocusedDepartmentsBranch[0].chiefId)}
+                    departmentAbbreviation={userFocusedDepartmentsBranch[0].abbreviation}
+                    treeLevel={0}
+                    onItemClicked = {this.props.onItemClicked}
+                />
+                {
+                    userFocusedDepartmentsBranch.map((head, index) => (
+                        <DepartmentsHScrollableList
+                            treeLevel={index + 1}
+                            departments={this.props.departments.filter(department => department.parentDepartmentId === head.departmentId)}
+                            departmentsLists={this.props.departmentLists[index + 1]}
+                            headDepartmentId={head.departmentId}
+                            headDepartmentChiefId={head.chiefId}
+                            focusOnDepartmentWithId={(index + 1) < userFocusedDepartmentsBranch.length ? userFocusedDepartmentsBranch[index + 1].departmentId : null}
+                            currentFocusedDepartmentId={this.props.currentFocusedDepartmentId}
+                            employees={this.props.employees}
+                            key={head.departmentId}
+                            updateDepartmentsBranch={this.props.updateDepartmentsBranch}
+                            requestEmployeesForDepartment={this.props.requestEmployeesForDepartment}
+                            onItemClicked={this.props.onItemClicked}
+                            employeesPredicate={(employee: Employee) => this.employeesPredicate(head, employee)}
```

Comment:
Every time a new lambda will be returned for the prop, and the component every time will be marked as _changed_. Move it to a separate method